auto_submit/lib/requests/check_pull_request.dart

```diff
@@ -65,6 +69,7 @@ class CheckPullRequest extends AuthenticatedRequestHandler {
         await pubsub.acknowledge('auto-submit-queue-sub', message.ackId!);
         continue;
       } else {
+        approver.approve(pullRequest);
```

Comment:
We should have an `await` to wait for the async function to finish.

Reply:
Nice catch, I wonder why the linter didn't complain about it. Updated.